client/src/Poem.jsx

```diff
@@ -15,6 +16,7 @@ const Poem = ({ poem }) => {
     const NBSP = '\u00A0'; // Unicode value for non-breaking space
     return <PoemLine line={(line.trim().length === 0) ? NBSP : line} key={index} />;
   });
+  const poemCommentsURL = '/api/comments/'.concat(poem.id);
```

Comment:
Using `+` caused [this linting error](http://eslint.org/docs/rules/prefer-template). Do you have objections to my using a template literal?